### PR TITLE
Fix broken API references in universe/python

### PR DESCRIPTION
**Pull request by NotTheEconomist**
Update some API references that had fallen out of use.

Notably this no longer ran, since docker.#Run now asks for an "input" not an "image," alpine.#Image became alpine.#Build and you have to reference its output. Additionally, fix up the FIXME tag in the simplest way possible, though a better solution may exist.

### CODE REVIEW

**Comment by aluzzardi** [comment]
Thank you for the contribution, @NotTheEconomist!

I think we should adopt the same pattern as `bash.#Run` (https://github.com/dagger/dagger/blob/main/pkg/universe.dagger.io/bash/bash.cue): use `core.#WriteFile` to write the file, rather than relying on `cat`.

The problem with `cat` is shell escaping which might cause problems with quotes, etc

**Comment by NotTheEconomist** [comment]
> The problem with `cat` is shell escaping which might cause problems with quotes, etc

Yeah I had a similar concern, but it was not immediately obvious how to do it another way. I'll use `bash.#Run` as a reference and update the PR a little later on today.

**Comment by NotTheEconomist** [comment]
Alright, try it now

**Comment by NotTheEconomist** [comment]
Fixed up the DCO just now, too.

**Comment by aluzzardi** [comment]
Great, looks good to me!

/cc @helderco @shykes for universe API change review

File: pkg/universe.dagger.io/python/python.cue
Comment (TomChv) [inline pkg/universe.dagger.io/python/python.cue:45]: What if someone is using `python2`,  will it works?

**Review by TomChv — CHANGES_REQUESTED** [review]
Nice contribution! I left few comments

File: pkg/universe.dagger.io/python/python.cue
Comment (NotTheEconomist) [reply pkg/universe.dagger.io/python/python.cue:52]: Yeah I could do that if it is preferred. The existing `python` package uses this method -- I only changed the alpine method.

It might be worthwhile to do that separately? This is a fix, that would be a refactor.

File: pkg/universe.dagger.io/python/test/test.cue
Comment (NotTheEconomist) [reply pkg/universe.dagger.io/python/test/test.cue:1]: Good eye, will fix :)

File: pkg/universe.dagger.io/python/python.cue
Comment (NotTheEconomist) [reply pkg/universe.dagger.io/python/python.cue:45]: No it won't, but also `python2` isn't being installed in the default image and hasn't been supported since 2020. I don't think we need to take pains to support it.

If users need support for Python 2 I could allow `command` to be overridden by the caller like `args` is and users can make their own #Image to pass in? Is that sufficient?

Comment (shykes) [reply pkg/universe.dagger.io/python/python.cue:45]: I think we can leave it as is, and if anyone complains about it, we can make it more extensible later.

File: pkg/universe.dagger.io/python/python.cue
Comment (shykes) [reply pkg/universe.dagger.io/python/python.cue:52]: This way is fine. We can refactor further later.

**Review by shykes — APPROVED** [review]
Thank you for this!

**Review by TomChv — APPROVED** [review]

**Comment by TomChv** [comment]
Clean up your commit history and I think it will be ready to be merge

**Comment by NotTheEconomist** [comment]
Whoa, I messed up that rebase, hold tight :)

**Comment by NotTheEconomist** [comment]
Nothing to see here, folks! Just a guy that's been working on Perforce projects for too long remembering how git works... :)

**Comment by aluzzardi** [comment]
> Nothing to see here, folks! Just a guy that's been working on Perforce projects for too long remembering how git works... :)

Haha no worries :)

I'm waiting for CI to pass then will merge.

Thanks again for your contribution, greatly appreciated!

**Comment by TomChv** [comment]
I started test, let's wait for a last approval of @helderco or @aluzzardi and if tests works, let's can merge it

**Review by aluzzardi — APPROVED** [review]

File: pkg/universe.dagger.io/python/python.cue
Comment (helderco) [reply pkg/universe.dagger.io/python/python.cue:45]: This should just be `python`.

**Review by helderco — APPROVED** [review]

File: pkg/universe.dagger.io/python/python.cue
Comment (NotTheEconomist) [reply pkg/universe.dagger.io/python/python.cue:45]: That actually fails. Alpine's python3 package does not symlink python to python3.

We could (and should! I may submit as another pr...) Use the python3-alpine image rather than building our own.

**Pull request merged**
